python/servo/bootstrap_commands.py

```diff
@@ -40,10 +40,6 @@ def env(self):
         env = self.build_env()
         print("export RUSTFLAGS=%s" % env["RUSTFLAGS"])
         print("export PATH=%s" % env["PATH"])
-        if sys.platform == "darwin":
-            print("export DYLD_LIBRARY_PATH=%s" % env["DYLD_LIBRARY_PATH"])
-        else:
-            print("export LD_LIBRARY_PATH=%s" % env["LD_LIBRARY_PATH"])
```

Comment:
Using shing's suggestion, we could use `env.get("LD_LIBRARY_PATH", "")` instead to avoid the error.